stylesheets/components/header.css

```diff
@@ -0,0 +1,78 @@
+.tr-header{
+    padding: var(--space-xs) var(--space-sm);
+    background-color: var(--tr-primary-color);
+    color: var(--tr-white);
+    font-size: var(--fs-md);
+    width: 100%;
+    align-items: center;
+    border-radius: 0 0 var(--space-md) var(--space-md);
+    gap: var(--space-sm);
+    flex-wrap: wrap;
+}
+
+.tr-heading h2{
+    font-size: var(--fs-xlg);
+    font-size: clamp(var(--fs-lg), -0.875rem + 8.33vw, var(--fs-xlg));
+}
+
+.tr-nav{
+    margin-left: auto;
+    flex-wrap: wrap;
+}
+
+.tr-nav a,
+.hamburger
+{
+    font-size: var(--fs-md);
+    color: var(--tr-light);
+    color: var(--tr-accent-color);
+    font-weight: 500;
+}
+
+.tr-nav a:hover i,
+.tr-nav a:focus i,
+.hamburger:hover i,
+.hamburger:focus i
+{
+    transition-delay: 0.1s;
+    color: var(--tr-primary-color);
```

Comment:
When you click any button in mobile, it will show an outline by default. To remove it add below code in focus property.
.btn:focus{
Outline: none;
}

Reply:
I have already removed outlines from all the button in the main `.tr-btn` class in the buttons component, so it never shows the default outline.